sdk/identity/identity/samples/samplefile.yml

```diff
@@ -0,0 +1,45 @@
+trigger:
```

Comment:
`samplefile.yml` and `image.png` are pretty generic filenames. Can we either:
- Name them appropriately or
- Place them under a directory called "azurePipelinesCredential"

We can still reference them if they're in a nested directory